openshift/olm/serverless-operator.v1.7.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,556 @@
+              labels:
+                name: knative-openshift
+                app: openshift-admission-server
+            spec:
+              serviceAccountName: knative-serving-operator
+              containers:
+                - name: knative-openshift
+                  image: $IMAGE_KNATIVE_OPERATOR
+                  command:
+                  - knative-openshift
+                  imagePullPolicy: Always
+                  env:
+                    - name: WATCH_NAMESPACE
+                      value: ""
+                    - name: POD_NAME
+                      valueFrom:
+                        fieldRef:
+                          fieldPath: metadata.name
+                    - name: OPERATOR_NAME
+                      value: "knative-openshift"
+                    - name: MIN_OPENSHIFT_VERSION
+                      value: "4.3.0-0"
+                    - name: REQUIRED_SERVING_NAMESPACE
+                      value: "knative-serving"
+                    - name: REQUIRED_EVENTING_NAMESPACE
+                      value: "knative-eventing"
+                    - name: KOURIER_MANIFEST_PATH
+                      value: deploy/resources/kourier/kourier-latest.yaml
+                    - name: CONSOLECLIDOWNLOAD_MANIFEST_PATH
+                      value: deploy/resources/console_cli_download_kn_resources.yaml
+                    - name: IMAGE_queue-proxy
+                      value: registry.svc.ci.openshift.org/openshift/knative-v0.13.2:knative-serving-queue
+                    - name: IMAGE_activator
+                      value: registry.svc.ci.openshift.org/openshift/knative-v0.13.2:knative-serving-activator
+                    - name: IMAGE_autoscaler
+                      value: registry.svc.ci.openshift.org/openshift/knative-v0.13.2:knative-serving-autoscaler
+                    - name: IMAGE_autoscaler-hpa
+                      value: registry.svc.ci.openshift.org/openshift/knative-v0.13.2:knative-serving-autoscaler-hpa
+                    - name: IMAGE_controller
+                      value: registry.svc.ci.openshift.org/openshift/knative-v0.13.2:knative-serving-controller
+                    - name: IMAGE_webhook
+                      value: registry.svc.ci.openshift.org/openshift/knative-v0.13.2:knative-serving-webhook
+                    - name: IMAGE_3scale-kourier-gateway
+                      value: docker.io/maistra/proxyv2-ubi8:1.0.8
+                    - name: IMAGE_3scale-kourier-control
+                      value: quay.io/3scale/kourier:fix_duplication
+                    - name: IMAGE_eventing-controller_eventing-controller
+                      value: registry.svc.ci.openshift.org/openshift/knative-v0.14.0:knative-eventing-controller
+                    - name: IMAGE_eventing-webhook_eventing-webhook
+                      value: registry.svc.ci.openshift.org/openshift/knative-v0.14.0:knative-eventing-webhook
+                    - name: IMAGE_broker-controller_eventing_controller
```

Comment:
@aliok 

is that `eventing_controller` rigt?  

should it be `eventing-controller` ?

Reply:
I think so: https://github.com/knative/eventing/blob/release-0.14/config/brokers/channel-broker/deployments/controller.yaml#L36

Reply:
I guess they changed that container name with 0.14, so we should change it in our image override too.

Reply:
@aliok no:

https://github.com/knative/eventing/blob/release-0.13/config/brokers/channel-broker/deployments/controller.yaml#L36